src/sql/util.py

```diff
@@ -447,22 +449,37 @@ def load_toml(file_path):
 def parse_toml_error(e, file_path):
     eline, ekey, evalue = get_line_content_from_toml(file_path, e.lineno)
     if "Duplicate keys!" in str(e):
-        return exceptions.ConfigurationError(f"Duplicate key found : '{ekey}'")
+        return exceptions.ConfigurationError(
+            f"Duplicate key found : '{ekey}' in {file_path}"
+        )
     elif "Only all lowercase booleans" in str(e):
         return exceptions.ConfigurationError(
-            f"Invalid value '{evalue}' in '{eline}'. "
+            f"Invalid value '{evalue}' in '{eline}' in {file_path}. "
             "Valid boolean values: true, false"
         )
     elif "invalid literal for int()" in str(e):
         return exceptions.ConfigurationError(
-            f"Invalid value '{evalue}' in '{eline}'. "
+            f"Invalid value '{evalue}' in '{eline}' in {file_path}. "
             "To use str value, enclose it with ' or \"."
         )
     else:
         return e
 
 
-def get_user_configs(file_path, section_names):
+def _display_configuration_message(prefix, html):
+    """Function to display message as HTML if HTML output
+    is supported in environment, e.g., Jupyter notebook else
+    in plain text"""
+    if html:
+        display.message_html(
+            f"{prefix}<a href='{CONFIGURATION_DOCS_STR}'>"
+            "configuration guideline</a>."
+        )
+    else:
+        display.message(f"{prefix}configuration guideline : {CONFIGURATION_DOCS_STR}")
```

Comment:
```suggestion
        display.message(f"{prefix}configuration guideline: {CONFIGURATION_DOCS_STR}")
```

no need to add a trailing space before the `:`

Reply:
[fixed](https://github.com/ploomber/jupysql/pull/823/commits/0859f7fe7b433027004c33630e024ca734621ecd)